conda/common/url.py

```diff
@@ -380,6 +382,29 @@ def remove_auth(url):
     return Url(**url_parts).url
 
 
+def escape_channel_url(channel):
```

Comment:
We could move this to `conda-libmamba-solver` if needed.

Reply:
This seems useful to me to keep in conda.